tasks.py

```diff
@@ -218,14 +219,27 @@ def deploy(ctx, space=None, branch=None, login=None, yes=False, migrate_database
     # Deploy API and worker applications
     for app in ('api', 'celery-worker', 'celery-beat'):
         deployed = ctx.run('cf app {0}'.format(app), echo=True, warn=True)
-        cmd = 'zero-downtime-push' if deployed.ok else 'push'
-        ctx.run('cf {cmd} {app} -f manifests/manifest_{file}_{space}.yml'.format(
+        cmd = 'push --strategy rolling' if deployed.ok else 'push'
+        # TODO: Test failing the build and rolling back
+        result = ctx.run('cf {cmd} {app} -f manifests/manifest_{file}_{space}.yml'.format(
             cmd=cmd,
             app=app,
             file=app.replace('-', '_'),
             space=space
         ), echo=True)
+        if not result.ok:
```

Comment:
Deploy is exiting itself - need to try to test partially failed build to see how to roll back before exiting the task